ViewDeck/IIViewDeckController.mm

```diff
@@ -239,6 +239,10 @@ - (void)openSide:(IIViewDeckSide)side animated:(BOOL)animated {
     [self openSide:side animated:animated notify:NO completion:NULL];
```

Comment:
Here we should now call `[self openSide:side animated:animated completion:NULL]` instead – same for the close... call.